tests/ballerina-integration-test/src/test/resources/jms/jmsservices/03_jms_map_message_subscriber.bal

```diff
@@ -26,12 +26,17 @@ service<jms:Consumer> jmsListener2 bind topicSubscriber2 {
 
     // OnMessage resource get invoked when a message is received.
     onMessage(endpoint subscriber, jms:Message message) {
-        map messageRetrieved = check message.getMapMessageContent();
-        io:print(messageRetrieved["a"]);
-        io:print(messageRetrieved["b"]);
-        io:print(messageRetrieved["c"]);
-        io:println(messageRetrieved["d"]);
-        byte[] retrievedBlob = check <byte[]>messageRetrieved["e"];
+        var messageContent = message.getMapMessageContent();
+        if (messageContent is map) {
+             map messageRetrieved = messageContent;
```

Comment:
Shall we open a jira for this. @SupunS please note. Since this operation looks redundant for me